package.json

```diff
@@ -25,7 +25,8 @@
     "eslint-plugin-spellcheck": "0.0.17",
     "intl": "~1.2.5",
     "jed": "^1.1.1",
-    "react-intl": "^2.8.0"
+    "react-intl": "^2.8.0",
+    "react-helmet": "^6.1.0"
```

Comment:
will be updated in foreman-js

Reply:
looks like `foreman-js` already updated to `6.1.0` https://github.com/theforeman/foreman-js/blob/master/packages/vendor-core/package.json#L73